awesomplete.js

```diff
@@ -201,15 +201,19 @@ _.prototype = {
 		}
 
 		if (selected) {
+			var suggestion = this.suggestions[this.index];
+
 			var allowed = $.fire(this.input, "awesomplete-select", {
-				text: this.suggestions[this.index],
+				text: suggestion,
 				origin: origin || selected
 			});
 
 			if (allowed) {
-				this.replace(this.suggestions[this.index]);
+				this.replace(suggestion);
 				this.close();
-				$.fire(this.input, "awesomplete-selectcomplete");
+				$.fire(this.input, "awesomplete-selectcomplete", {
+					text: suggestion
```

Comment:
Should we add `origin: origin || selected` too as in `awesomplete-select` above?